### ISSUE TITLE
E116: Invalid arguments for function SpaceVim#logger#info with nvim in diff mode

### ISSUE THREAD

**Issue by Antalis** [wsdjeg/SpaceVim#4802]
## With Spacevim installed, the "nvim -d" command produces the following error:

```
Error detected while processing /home/spacevim/.SpaceVim/init.vim[58]..function SpaceVim#begin:
line   75:
E684: list index out of range: 1
E116: Invalid arguments for function string
E116: Invalid arguments for function SpaceVim#logger#info
Press ENTER or type command to continue
```

## The reproduce ways from Vim starting (Required!)
Under GnuLinux (in my case Fedora 37 up to date : python3-neovim-0.4.3-7.fc37.noarch & neovim-0.8.3-1.fc37.x86_64
):
- create a new user account and log in
- run SpaceVim install (curl -sLf https://spacevim.org/install.sh | bash)
- Install and update default plugins.
- run "nvim -d"

## Output of the `:SPDebugInfo!`
`SPDebugInfo!` does not work with nvim in diff mode

## Screenshots

If you have any screenshots for this issue, please upload here. BTW you can use https://asciinema.org/ for recording video in terminal.

**Comment by twio142** [comment]
Same problem here

**Comment by wsdjeg** [comment]
Please update spacevim, I can not reproduce this issue.

**Comment by Antalis** [comment]
The problem seems to have been solved with the commit #4782.
Thx
